torch/_vmap_internals.py

```diff
@@ -158,9 +158,10 @@ def vmap(func: Callable, in_dims: in_dims_t = 0, out_dims: out_dims_t = 0) -> Ca
     gradients when composed with autograd.
 
     .. note::
-        We are actively developing a different and improved vmap prototype
-        `here. <https://github.com/zou3519/functorch>`_ The improved
-        prototype is able to arbitrarily compose with gradient computation.
+        We have moved development of vmap to
+        `functorch. <https://github.com/pytorch/functorch>`_ functorch's
+        vmap is able to arbitrarily compose with gradient computation
+        and contains significant performance improvements.
         Please give that a try if that is what you're looking for.
 
         Furthermore, if you're interested in using vmap for your use case,
```

Comment:
Do we want to update this to send people to functorch instead so things don't get lost in a pytorch issue?

Reply:
That pytorch issue directs folks back to functorch (and also has instructions for how to find us on Slack) so I think it's fine to leave it for now.